### PR TITLE
[element 2.0] Support browserLaunchOptions in TestSetting

### PR DESCRIPTION
**Pull request by phileba**
resolve https://tricentisflood.atlassian.net/browse/FLO-2004

### CODE REVIEW

File: packages/core/src/page/types.ts [hunk 1]
```diff
@@ -1,11 +1,7 @@
 import { ElementHandle as PElementHandle, Frame, Page } from 'playwright'
 import { Point } from './Point'
 
-export enum BROWSER_TYPE {
-	CHROME = 'chromium',
-	FIREFOX = 'firefox',
-	WEBKIT = 'webkit',
-}
+export type BrowserType = 'chromium' | 'firefox' | 'webkit'
```
Comment (ivanvanderbyl) [inline packages/core/src/page/types.ts:4]: Should this include 'edge' ?

Comment (phileba) [reply packages/core/src/page/types.ts:4]: they are 3 types were supported by playwright, to run with Edge, the user should be defined in `TestSetting`
```ts
export const settings: TestSettings = {
	loopCount: 1,
	browserLaunchOption: {
		executablePath: '/Applications/Microsoft Edge.app/Contents/MacOS/Microsoft Edge',
		headless: false,
	},
}
``` 
we can not handle the path of the Edge in cross-platform cause of some permission, so let the user do it.

File: packages/core/src/runtime/Test.spec.ts [hunk 2]
```diff
@@ -5,10 +5,9 @@ import Test from './Test'
 import { EvaluatedScript } from './EvaluatedScript'
 import { join } from 'path'
 import { EventEmitterReporter } from '@flood/element-report'
-import { ConcreteTestSettings, normalizeSettings, TestSettings } from './Settings'
```
Comment (honglax) [inline packages/core/src/runtime/Test.spec.ts:8]: Is TestSettings redundant?

Comment (phileba) [reply packages/core/src/runtime/Test.spec.ts:8]: yep @honglax

**Review by honglax — APPROVED** [review]
LGTM 🚀

**Pull request merged**